python-sdk/src/astro/databases/snowflake.py

```diff
@@ -766,6 +757,52 @@ def _build_merge_sql(
         }
         return statement, params
 
+    def append_table(
```

Comment:
Why are we adding this function?

Reply:
This is just isolation from other databases. Since adding quoted was failing for bigquery.